### PR TITLE
SC-4520 - activation service

### PR DESCRIPTION
**Pull request by BlackGhostHD**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-4520

## Screenshots of UI changes

Opera Momentaufnahme_2020-06-30_114419_localhost
Opera Momentaufnahme_2020-06-15_104102_localhost

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5696246&bid=18643699)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5696246&bid=18643699)

File: controllers/account.js [hunk 1]
```diff
@@ -75,6 +75,33 @@ router.get('/', (req, res, next) => {
 	});
 });
 
+router.get('/teams/', (req, res, next) => {
+	const isDiscoverable = res.locals.currentUser.discoverable;
+
+	res.render('account/teams', {
+		title: 'Team Einstellungen',
```
[the remark this answers is not in the document]
Comment (BlackGhostHD) [reply controllers/account.js]: done :)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5696246&bid=18643699)

**Pull request merged**